lib/urlutil.js

```diff
@@ -65,7 +78,11 @@ const TPL_TRANSFORMS = {
       return '';
     });
   },
-  randomint: (uuid, params) => {
-    return Math.floor(Math.random() * MAXINT);
+  url: (path, p) => {
+    let u = `${p.host}/${p.program}${p.path}`;
```

Comment:
`path` in my limited testing usually starts with a `\`, so I don't add one in between here.

Reply:
also, I went back to look at their examples, and the podtrac requested value does not start with a protocol, so I am not using it here, but I do need the rest of it

Reply:
Err... looking in BQ, it seems like paths do _not_ start with a slash.  Maybe best to just handle both here?